website/src/contribute/postcommits-policies.md

```diff
@@ -65,11 +65,34 @@ When a post-commit test fails, follow the provided steps for your situation.
 
 ### My change was rolled back due to a test failure {#pr-rolled-back}
 
-1.  Look at the JIRA issue to find the reason for the rollback.
-1.  Fix your code and re-run the post-commit tests.
-1.  Implement new pre-commit tests that will catch similar bugs before future
-    code is merged into the repository.
-1.  Open a new PR that contains your fix and the new pre-commit tests.
+After rollback there is time for deeper investigation. Start by looking at the
+JIRA issue to see the background information for the rollback. There are three
+common scenarios:
+
+1.  Your change contained a bug.
+2.  Your change exposed an existing bug.
+3.  Your change exposed a bad test (flaky, overspecified, etc).
+
+_These are all valid reasons for rollback. Maintaining clear signal is the
+highest priority._
+
+The high level steps are the same:
+
+1.  Create a fix and re-run the post-commit tests.
+2.  Implement new pre-commit tests that will catch similar failures
+    before future code is merged into the repository.
+3.  Open a new PR that contains your fix and the new pre-commit tests.
+
+If the bug is not in your code, here is how to "create a fix":
+
+1.  File a ticket for the existing bug, if it does not already exist.
+    Remember that
+    [a flaky test is a critical bug]({{ site.baseurl }}/contribute/postcommits-policies-details/index.html#flake_is_failing). Other
+    bad tests are similar: they may fail for arbitrary reasons having nothing
+    to do with what it is testing, making our signal unreliable.
+2.  Keep a sickbayed copy of the test that reproduces the failure.
```

Comment:
1. Is *sickbayed* is common terminology? ([Google didn't find it](https://www.google.com/search?ei=g2zXW6bICvOBk-4P8pSNoA4&q=software+sickbay+test&oq=software+sickbay+test)) If you want to use it, consider defining it first or linking to a definition. Or reword this.

2. I'm not sure what you mean in steps 2 and 3: 

* What does this mean to "keep a sickbayed copy of the test"? Like the keep the source but `@Ignore` it? Or capture the link to a failed Jenkins job?
* What are you suggesting the new test should be? Maybe you mean if an existing test is overspecified, create a new test based on the existing with the correct specification?

Here's my best shot at a suggested rewording, apologies if I've missed some of your intent:

```
1.  File a ticket for the existing bug, if it does not already exist.
    Remember that
    [a flaky test is a critical bug]({{ site.baseurl }}/contribute/postcommits-policies-details/index.html#flake_is_failing). Other
    bad tests are similar: they may fail for arbitrary reasons having nothing
    to do with what it is testing, making our signal unreliable.
2.  Mark the problematic test to be skipped, with a link to the JIRA ticket
3.  (Preferred) Update or create a new test that fixes the issue.
```

Reply:
Your phrasing captures it much better than mine did. I'm trying to get at the fact that the failing test might have some value that can be preserved even as the exact thing has to be disabled. But, indeed, that is too fine a distinction to draw. In fact, I think even the "preferred" path is not reasonable. There's a fine chance that the contributor does not have the context to actually fix the thing.

Incidentally, we have a `sickbay` tag in Beam that we might want to link to somewhere to explain. Probably a matter for the dev wiki.